baseplate/frameworks/thrift/__init__.py

```diff
@@ -57,7 +57,6 @@ def call_with_context(*args: Any, **kwargs: Any) -> Any:
                 span.set_tag("exception_type", "Error")
                 span.set_tag("thrift.status_code", exc.code)
                 span.set_tag("thrift.status", status)
-                span.set_tag("success", "false")
```

Comment:
Doesn't this still need to be set for 500-599?

Reply:
No it happens automatically because 5xx use the `span.finish(exc_info=sys.exc_info())` call which set `success` to `False` based on whether `exc_info` is passed or not.